### PR TITLE
Update README.md

### LINKED ISSUE #545: [OTHER] <Same resource link multiple times>
### Which field of community (DevOps/Opensource/Web/Web3/Android/Devrel) you want to add, drop the links?

Hey there! In DevOps roadmap, the containers resource section has same youtube link twice. Please remove it.

Thankyou!
Same

### PR DESCRIPTION
**Pull request by VishwaS-22**
## Fixes Issue

Closes #545

## Changes proposed

Removed duplicate link.

Happy to contribute. Thankyou!

### CODE REVIEW

File: DevOps/README.md [hunk 1]
```diff
@@ -58,7 +58,6 @@
   - [Docker Tutorial - TechWorld with Nana(Youtube)](https://www.youtube.com/watch?v=3c-iBn73dDE)
   - [Learn Docker - freecodecamp(Youtube)](https://www.youtube.com/watch?v=9zUHg7xjIqQ)
   - [Docs](https://docs.docker.com/)
-  - [Complete Docker Course](https://www.youtube.com/watch?v=RqTEHSBrYFw)
   - [Complete Docker Course - From BEGINNER to PRO! (Learn Containers)](https://www.youtube.com/watch?v=RqTEHSBrYFw&t=3615s)
```
Comment (siddhant-khisty) [inline DevOps/README.md:61]: Please add the channel name as well, as that we can avoid duplicate resources in the future.
```suggestion
  - [Complete Docker Course - From BEGINNER to PRO! (Learn Containers) - by DevOps Directive](https://www.youtube.com/watch?v=RqTEHSBrYFw&t=3615s)
```

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]

**Comment by VishwaS-22** [comment]
Hey @siddhant-khisty! Updated the requested changes!
Thankyou!

**Review by Sandeep-RN — APPROVED** [review]
LGTM

**Review by siddhant-khisty — APPROVED** [review]
lgtm

**Comment by VishwaS-22** [comment]
Hey @siddhant-khisty! Thanks for merging it. 

Happy to contribute to the community.

**Pull request merged**